src/App.js

```diff
@@ -11,30 +15,33 @@ function App() {
 	const [products] = useState(data);
 	const [cart, setCart] = useState([]);
 
+	// MAKING FUNCTION TO ALLOW USER TO ADD ITEMS TO CART
+		// NEED TO MAKE AN OBJECT WITH THE PROPERTIES THAT THIS NEW ITEM WILL HAVE <NEWITEM>
+		// SETCART AS WHATEVER IS IN THE CARD PLUS THE NEW ITEM WE ADD TO IT
 	const addItem = item => {
-		// add the given item to the cart
+		setCart([...cart, item]);
 	};
 
+	const removeItem = item => {
+		const deletedItem = cart.filter(element => {
+			return item !== element.id
+		})
+			setCart(deletedItem)
+	}
+
```

Comment:
Removing the item from the list 💪 

The naming convention in this case is a little strange because `deletedItem` is an array of all items _minus_ the deleted one - what could be a good name for this variable?